code/game/machinery/computer/orders/order_items/mining/order_pka.dm

```diff
@@ -29,6 +29,14 @@
 	item_path = /obj/item/gun/energy/recharge/kinetic_accelerator/m79
 	cost_per_order = 1250
 
+/datum/orderable_item/accelerator/mag //monke edit
```

Comment:
all of these monke-added datums should be in the `monkestation/` modular directory